README.md

```diff
@@ -26,23 +26,21 @@ While this can be achieved by simply exposing a `/status` endpoint that perfoms
 
 With that said, not everyone _needs_ asynchronous checks. If your service has one dependency (and that is unlikely to change), it is trivial to write a basic, synchronous check and it will probably suffice.
 
-However, if you anticipate that your service will have several dependencies, with varying degrees of complexity for determing their health state - you should probably think about introducing asynchronous health checks.
+However, if you anticipate that your service will have several dependencies, with varying degrees of complexity for determining their health state - you should probably think about introducing asynchronous health checks.
 
 ## How does this library help?
-Writing an async healthchecking framework for your service is not a trivial task, especially if Go is not your primary language.
+Writing an async health checking framework for your service is not a trivial task, especially if Go is not your primary language.
 
 This library:
 
 * Allows you to define how to check your dependencies.
 * Allows you to define warning and fatal thresholds.
 * Will run your dependency checks on a given interval, in the background. **[1]**
 * Exposes a way for you to gather the check results in a *fast* and *thread-safe* manner to help determine the final status of your `/status` endpoint. **[2]**
-* Comes bundled w/ [pre-built checkers](/checkers) for well-known dependencies such as `Redis`, `HTTP`.
+* Comes bundled w/ [pre-built checkers](/checkers) for well-known dependencies such as `Redis`, `Mongo`, `HTTP` and more.
 * Makes it simple to implement and provide your own checkers (by adhering to the checker interface).
-* Is test-friendly
-    + Provides an easy way to disable dependency health checking.
-    + Uses an interface for its dependencies, allowing you to insert fakes/mocks at test time.
-* Allows you to trigger listener functions when a health check fails or recovers. **[3]**
+* Allows you to trigger listener functions when your health checks fail or recover using the `IStatusListener` interface.
```

Comment:
Link `IStatusListener interface` to the `IStatusListener` section further down in the README.

Same for the OnComplete bullet.